### PR TITLE
Add debug logs

### PR DESCRIPTION
**Pull request by janisz**
## Description

This PR add couple of debug log statements to narrow down issue with missing deployment.
We cannot reproduce it so we hope we get more logs if it fails on nightly or other PR.
Refs: https://github.com/stackrox/stackrox/pull/2661
## Testing Performed

CI

Refs:  ROX-12096

### CODE REVIEW

**Comment by janisz** [comment]
Current dependencies on/for this PR:
* master
  * **PR #2994** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/2994" target="_blank">Graphite</a>  👈
    * **PR #2995** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/2995" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/stackrox/stackrox/2994?utm_source=stack-comment).

File: central/risk/manager/manager.go [hunk 1]
```diff
@@ -145,6 +148,7 @@ func (e *managerImpl) ReprocessDeploymentRisk(deployment *storage.Deployment) {
 	}
 
 	if oldScore == risk.GetScore() {
+		log.Debugf("Old riks equals existing risk, meaning we do not upsert for deployment %q", deployment.GetName())
```
Comment (gavin-stackrox) [inline central/risk/manager/manager.go:151]: nit:
```suggestion
		log.Debugf("Old risk equals existing risk, meaning we do not upsert for deployment %q", deployment.GetName())
```

**Review by gavin-stackrox — COMMENTED** [review]
It looks OK to me but I don't have any insight into potential performance impacts of debug statements in central so I'll leave it with you guys to approve.

**Review by dhaus67 — COMMENTED** [review]
How about doing the same conditional log like before, so we only log the sac deployment? Otherwise the log will be quite big.

Additionally, let's wait for the first RC to be created before merging this one.

**Comment by janisz** [comment]
> How about doing the same conditional log like before, so we only log the sac deployment? Otherwise the log will be quite big.

But then we will wait on repro only in this one test we spotted it. What if it happens in other tests?

**Comment by janisz** [comment]
@dhaus67 @gavin-stackrox PTAL

**Comment by janisz** [comment]
@dhaus67 Good point. Maybe we can enable logs for module

**Comment by janisz** [comment]
@dhaus67 How about it now? Added filtering and keep everything in single function so it should be easy to remove.

**Review by janisz — COMMENTED** [review]
/restest

**Review by dhaus67 — APPROVED** [review]

**Pull request merged**